code/modules/projectiles/updated_projectiles/guns/shotguns.dm

```diff
@@ -217,12 +217,12 @@ can cause issues with ammo types getting mixed up during the burst.
 
 
 /obj/item/weapon/gun/shotgun/combat/set_gun_config_values()
-	fire_delay = config.mhigh_fire_delay*2
-	accuracy_mult = config.base_hit_accuracy_mult
-	accuracy_mult_unwielded = config.base_hit_accuracy_mult + config.low_hit_accuracy_mult - config.hmed_hit_accuracy_mult
+	fire_delay = config.high_fire_delay * 3 //one shot every 1.5 seconds.
+	accuracy_mult = config.base_hit_accuracy_mult + config.low_hit_accuracy_mult
```

Comment:
Do document buffs you make. Don't do stealth buffs.

Reply:
It's just copy pasta